docs/language/usage.rst

```diff
@@ -51,12 +51,13 @@ or pass in ``credentials`` explicitly.
 Documents
 *********
 
-The Google Natural Language API has three supported methods
+The Google Natural Language API has five supported methods
```

Comment:
Instead of changing the number, drop it?

> The Google Natural Language API supports the following methods:

Reply:
Probably a good idea. Will do in a followup.

Reply:
Nevermind. I had to fix docs anyway (I made a mistake that broke the build) so I updated this.